rust/config/mainnet_config.json

```diff
@@ -128,6 +128,20 @@
       "index": {
         "from": 25900000
       }
+    },
+    "zkevm": {
```

Comment:
Does this set a weird precedent? We don't really want folks to be adding their config here right? I was imagining that we would track this config somewhere in `infra` to signify that

Reply:
I don't think we have a good answer, and this was easiest

Reply:
Why can't we just track this config in `infra` and then insert it via helm?